src/scripts/inputManager.js

```diff
@@ -22,9 +22,7 @@ define(['playbackManager', 'focusManager', 'appRouter', 'dom'], function (playba
 
     var eventListenerCount = 0;
     function on(scope, fn) {
-        if (eventListenerCount) {
-            eventListenerCount++;
-        }
+        eventListenerCount++;
```

Comment:
I seem to recall this being the cause of a bug at some point.

Reply:
With condition, subscription is useless - `eventListenerCount` is always "0".
I think the goal of the counter is not to spam events if there are no listeners.
https://github.com/jellyfin/jellyfin-web/blob/ec6ce5aa14384f51ff4f97a749bb4e0985aed773/src/scripts/inputManager.js#L63-L77
Possible flaws:
* subscribers do not unsubscribe
* subscribers unsubscribe more than they subscribe
* some subscriber mistakenly cancels event